command/s3_put.go

```diff
@@ -137,6 +140,9 @@ func (s3pc *s3put) validate() error {
 	if s3pc.isMulti() && filepath.IsAbs(s3pc.LocalFile) {
 		catcher.Add(errors.New("cannot use absolute path with local_files_include_filter"))
 	}
+	if s3pc.Visibility == "signed" && (s3pc.Permissions == "public-read" || s3pc.Permissions == "public-read-write") {
+		catcher.Add(errors.New("visibility: signed should be combined with permissions: private"))
```

Comment:
might it be confusing if we direct the user to use specifically private, but other options are also okay (such as `bucket-owner-read`)?